kymatio/scattering2d/tests/test_numpy_scattering2d.py

```diff
@@ -149,10 +149,14 @@ def test_fft(self, backend):
 
         z = backend.fft(x, direction='C2C', inverse=True)
 
+        z = z * 4
```

Comment:
i would prefer if you didn't update the z between the definition and the test

Reply:
@janden see